### ISSUE TITLE
[20.09] Network manager crash in i3gaps

### ISSUE THREAD

**Issue by lukasz-m-maciejewski** [NixOS/nixpkgs#108624]
**Describe the bug**
Duplicate of  [#99197](https://github.com/NixOS/nixpkgs/issues/99197), but on the i3gaps fork of the i3 WM.

Any tray app ( Network Manager applet in particular) crashes after an in-place restart of i3 or e.g. after resume from suspend.

**To Reproduce**
1.
```
programs.nm-applet.enable = true
services.xserver.windowManager.i3.enable = true;
services.xserver.windowManager.i3.package = pkgs.i3-gaps 
```
2. Log in to the session and restart i3 in place.

**Expected behavior**
Tray applications persist after restart.

**Metadata**
- system: `"x86_64-linux"`
 - host os: `Linux 5.4.86, NixOS, 20.09.2497.4a75ca4a4e7 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.9`
 - channels(root): `"nixos-20.09.2497.4a75ca4a4e7"`
 - channels(lukaszm): `"home-manager-20.09"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
@fmthoma 
```yaml
# a list of nixos modules affected by the problem
module: i3-gaps
```

**Comment by thiagokokada** [comment]
Probably the same fix from i3 can be applied here, backport this commit to 20.09: https://github.com/NixOS/nixpkgs/commit/9190daba45c50e7c704aedcfb7438ef4938c0343#diff-7cb3a901a6ef0400193cdd9612a05b4fde580d93eeec2a1d2817b29596e7389d

The issue seems to be version 4.18.2, 4.18.3 should be fine.

**Comment by thiagokokada** [comment]
Opened a PR: https://github.com/NixOS/nixpkgs/pull/108871

**Comment by timokau** [comment]
Should be fixed by #108871. Please test and re-open if the issue persists. You can check [status.nixos.org](https://status.nixos.org/) to see when the `nixos-20.09` channel advances. You can also rebuild your system from a local nixpkgs checkout if you do not want to wait for the channel.
